src/interfaces/UploadOptions.ts

```diff
@@ -1,3 +1,25 @@
+interface TransformationObject {
+  type: "transformation";
+  value: string;
+}
+
+interface GifToVideoOrThumbnailObject {
+  type: "gif-to-video" | "thumbnail";
+  value?: string;
+}
+
+interface AbsObject {
+  type: "abs";
+  value: string;
+  protocol: "hls" | "dash";
+}
+
+type postTransformation = TransformationObject | GifToVideoOrThumbnailObject | AbsObject;
```

Comment:
Interfaces should be named with proper CamelCasing

Reply:
done